include/nrf_rpc/rpc_utils/dev_info.h

```diff
@@ -18,13 +18,27 @@ extern "C" {
  * @{
  */
 
+ struct nrf_rpc_crash_info
```

Comment:
Why is this part of `dev_info` library? The crash dump retrieval has been implemented in `log_rpc` module, so it would make sense to put the function for retrieving the dump summary there as well, right?